community/sway/etc/sway/definitions

```diff
@@ -60,11 +58,10 @@ set $shutdown wlogout --buttons-per-row 3
 set $pulseaudio $term_float pulsemixer
 
 # help command
-set $help $term_float "man $HOME/.config/sway/help.man"
+set $help "$term_float man $HOME/.config/sway/help.man"
 
 # onscreen bar
-set $onscreen_bar bash /usr/share/sway/scripts/wob.sh
-
+set $onscreen_bar bash /usr/share/sway/scripts/wob.sh "$accent-colorFF" "$background-colorFF" 
```

Comment:
does sway escape that correctly? i'd have assumed that it searches for a variable `accent-colorFF`. interesting this works

Reply:
It does, and I learned it the hard way, while using $term-font, it gave me `foot-font` and not `Roboto NerdFonts`. :|

Reply:
It does match for as much as possible while within quotes though.

Reply:
irritating, but ok.